doc/api/test.md

```diff
@@ -3030,6 +3128,59 @@ test('top level test', async (t) => {
 });
 ```
 
+### `context.assert`
+
+<!-- YAML
+added:
+  - v22.2.0
+-->
+
+An object containing assertion methods bound to `context`. The top-level
+functions from the `node:assert` module are exposed here for the purpose of
+creating test plans.
+
+```js
+test('test', (t) => {
+  t.plan(1);
+  t.assert.strictEqual(true, true);
+});
+```
+
+#### `context.assert.snapshot(actual[, options])`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1.0 - Early development
+
+* `actual` {any} A value to serialize to a string. If Node.js was started with
+  the [`--test-update-snapshots`][] flag, the serialized value is written to
+  the snapshot file. Otherwise, the serialized value is compared to the
+  corresponding value in the existing snapshot file.
+* `options` {Object} Optional configuration options. The following properties
+  are supported:
+  * `serializers` {Array} An array of synchronous functions used to serialize
```

Comment:
Any reason this is an array and not a single function? Why would I call multiple serializers?

Reply:
It's just a bit more flexible without really adding complexity. For example if you want to serialize something using `util.inspect()`, then normalize line endings across platforms, then do something else. You could put all of that logic in a single function, but it might be easier to reason about (and reuse code) if you can use multiple functions.

Reply:
I'm +1 with starting with an array, we can improve DX later